docs/antora-playbook-local.yml

```diff
@@ -0,0 +1,19 @@
+# Use this configuration for local testing / docs develop.
+site:
+  title: TMail Documentation
+  url: https://documentation.tmail.org/
+  start_page: tmail-site::index.adoc
+content:
+  sources:
+    - url: ./..
+      branches: HEAD
+      start_path: james-project/docs
```

Comment:
How that allows you to combine it with the Apache James Antora doc? That's sweet :)